tests/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -105,9 +116,10 @@ TEST (nnstreamer_capi_playstop, dummy_01)
   EXPECT_NE (state, ML_PIPELINE_STATE_UNKNOWN);
   EXPECT_NE (state, ML_PIPELINE_STATE_NULL);
 
-  g_usleep (150000); /* 50ms is good for general systems, but not enough for emulators to start gst pipeline. Let a few frames flow. */
```

Comment:
@myungjoo , Is This PR to handle QEMU-Based ARM as well as (hig-performance) X86 for runtest? What is the meaning of the "**in slow systems**" that is described in the PR title? NOTE that the system freeze issue of the unit test in Tizen/GBS can not be completely solved with the short sleep time. (In case of Tizen/GBS based on QEMU/ARM), The issue is generated due to slow software IO emulation as well as slow CPU operation of a process.

Reply:
**slow systems** means any systems that may incur more than 50ms to configure a pipeline.